XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectPatchPage.cs

```diff
@@ -255,7 +253,15 @@ private void PageLeaveCancelled(string message)
         private void PopulatePatchesBox()
         {
             dataGridViewPatches.Rows.Clear();
-            var updates = new List<Alert>(Updates.UpdateAlerts);
+
+            var alerts = Updates.UpdateAlerts.ToList();
+            if (dataGridViewPatches.SortedColumn == null)
+            {
+                alerts.Sort(new NewVersionPriorityAlertComparer());

```

Comment:
Can this not be done in the `else` block of the `if (dataGridViewPatches.SortedColumn != null)` below?